test/test-applications/integrations/Samples.Http/Directory.Build.props

```diff
@@ -0,0 +1,11 @@
+<Project>
+  <PropertyGroup>
+
+    <OutputType>Exe</OutputType>
+    <TargetFrameworks>netcoreapp3.1;net5.0;net6.0</TargetFrameworks>
```

Comment:
Would it be enought to remove this fine and set `<TargetFrameworks>netcoreapp3.1;net5.0;net6.0</TargetFrameworks>` in `Samples.Http.csproj` ?